sass/components/_file.scss

```diff
@@ -225,29 +225,76 @@ Style guide: file.3
 */
 .file--link {
   background-color: transparent;
+  @extend .col-md-10;
+  @extend .col-sm-10;
+  @extend .col-xs-10;
+  border-top: 1px solid $ec-grey15;
+  &:last-child {
+    border-bottom: 1px solid $ec-grey15;
+  }
+  padding: 1em 0;
+  margin-bottom: 0;
+  .file__title {
 
-  .file__metadata--link {
-    @extend .col-md-10;
-    @extend .col-sm-10;
-    @extend .col-xs-10;
+    &:link {
+      text-decoration: none;
+    }
+    &:hover,
+    &:active {
+      text-decoration: underline;
+    }
 
-    margin-bottom: 0;
-    padding-left: 0;
-    padding-right: 0;
-    line-height: 1;
-  }
+    &::after {
+      padding-left: .4em;
+      margin-left: .4em;
+      border-left: 2px solid $ec-grey50;
+      color: $ec-grey;
+
+      // This is needed to have the text-decoration working.
+      display: inline-block;
+      text-decoration: none;
+    }
+    &::before {
+      @extend .icon;
+      color: $ec-grey50;
+      font-size: 1.4em;
+      display: table-cell;
+      vertical-align: bottom;
+      width: 40px;
+      float: left;
+    }
 
-  .icon--file-link {
```

Comment:
Doing this we only set the icon for the recognized extensions, but we should have a default, no? I suppose that the default is the pdf icon, btw...